app.rb

```diff
@@ -0,0 +1,58 @@
+require './book'
+require './student'
+require './teacher'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def create_student(age, name, parent_permission)
+    classroom = 'Null'
+    @people << { type: 'Student', data: Student.new(age, name, classroom, parent_permission) }
+  end
+
+  def create_teacher(age, name, spec)
+    @people << { type: 'Teacher', data: Teacher.new(age, spec, name) }
+  end
```

Comment:
## The same issue here when I am adding a new teacher ⬇⬇
- [ ] Great job so far  but I think you have an error here you are passing a wrong number of arguments (given 3, expected 1..2),  so when I am try to create a new teacher I can't ceate it because there is not coordinates with the given arguments so please fix this error to let the user be able to create a new teacher 💯😊📌🔥♻✅

## this is the given error ♻📌🔥💯💯


![image](https://user-images.githubusercontent.com/58553711/216479848-22be3798-4058-4c01-b930-ee79b2ff018c.png)